src/app/core/header/header.component.spec.ts

```diff
@@ -22,4 +23,22 @@ describe('HeaderComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+  
+  it('should call exit', () => {
+    let spy = spyOn(component, 'exit');
+
+    let button = fixture.debugElement.nativeElement.querySelector("img[src='assets/img/ic-exit.svg']");
+    button.click();
+
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should log on exit', () => {
+    const spy = spyOn(console, 'log');
+
+    let button = fixture.debugElement.nativeElement.querySelector("img[src='assets/img/ic-exit.svg']");
```

Comment:
Consider adding a specific class to `img` tag, in that case, we will not tie to the source link, so we will be more flexible.

Reply:
Done